### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by ghost** [STR0UDY/wf_alerts#1]
adding this allowing people to click a button and select the waypoint on each notification would be awesome.

https://clips.twitch.tv/SmellyLongElephantBrokeBack

**Comment by STR0UDY** [comment]
Oh nice, yeah I'll definetly look into doing something similiar, you'll have to bear with me though I've been busy lately.

**Comment by ghost** [comment]
No rush dude i appreciate your response thanks for the great script in the first place.

**Comment by STR0UDY** [comment]
I’d say ETA late tomorrow, I won’t close this until it’s pushed anyway.

**Comment by ghost** [comment]
thanks man i appreciate your work

**Comment by STR0UDY** [comment]
Been working on this, practically doing a rewrite to be a bit more like Koils, I'll explain in full detail when it's done but because of this essentially being a rewrite on the back end it's gonna take another day or so, been super busy due to this whole covid-19 and what not.

**Comment by ghost** [comment]
No rush dude take your time yeah this covid sittuation sucks

i appreciate you doing this for the community man.

**Comment by STR0UDY** [comment]
Adding the first part where it uses database entry and it's a bit more cleaner imo, the logging will come in a few more days, just been super busy. Sorry 😅 

You can find the new version inside the [WIP] folder, with an included how to use text document.

**Comment by AleDema** [comment]
Out of curiosity why have you decided to save the alert data on the database instead of using a cfg?

**Comment by STR0UDY** [comment]
Koil uses a database and imports that database into a table, I can think of a few reasons
- Performance
- If you're doing development work and streaming and don't want to show off all your codes and let it easily get copied
- Ease of Access

He might of not done it for any of them reasons, but koil is an amazing coder so I'd like to think theres a good reason thus my reasoning for doing it.

**Comment by ghost** [comment]
Great work dude looking forward to the part where we can scroll and put the marker on the map from the list of alerts

**Comment by AleDema** [comment]
Personally I don't think there are any performance gains using a database over a cfg, if anything its slower (accessing memory is faster than querying the db), I may be biased but I think using a cfg is also easier. Koil is a skilled coder, but I wouldn't look into his design decisions too much, from the look of it NP's code is quite messy and lots of stuff is written on the fly without putting too much thought into it. Imho the DB should be used only to store variable data that needs to be persisted across sessions, if you have static data there are no reasons to store them into the db, ESX used this approach for lots of stuff and it bit them in the ass, cause when the tables get large enough Fivem's messagepack can't serialize them anymore, this shouldn't realistically happen in your case, but its still worth noting.

**Comment by STR0UDY** [comment]
Yeah of course, that's very correct, Of course they can be moved into a standard table, at the moment the main reason is because I honestly don't want to write them one-by-one into a table which is just laziness on my side which I won't lie, a quick look through this script you'll see where I got lazy which is something I intend to improve on, so the resource can actually be a "resource" rather than a _resource_ if you catch my drift, but thanks for raising you opinions and it won't go un-noticed. 😄

**Comment by AleDema** [comment]
Don't worry you're doing an amazing job, I was just curious on why you decided to go this route, maybe there was something I was missing. Anyway keep up the good work :D

**Comment by ghost** [comment]
Any word on this str0udy?

**Comment by STR0UDY** [comment]
Honestly I been lazy lately, focusing on other stuff but I'll definetly try get this pushed out soon, the problem with when I write resources is that I tend to overwrite and then want to rewrite and before you know it, I've rewritten the entire resource.

**Comment by AleDema** [comment]
@ONYX-X How does it work on NP? Do they press a key to enable the nui focus?

**Comment by ghost** [comment]
they do

**Comment by DRIFTKINGEVO** [comment]
Any news? :)
